src/components/extractFields.js

```diff
@@ -10,11 +10,16 @@ function isUsCounty(base_record, qs_a2_alt) {
 
 // this function favors gn:population when available, falling back to zs:pop10
 //  when available and > 0
+// the mz:population should be moved to the top when implemented
```

Comment:
If `mz:population` is moved to the top now then we won't have to change the code when it's supported.